.buildkite/distributed/pipeline.yml

```diff
@@ -51,7 +51,7 @@ steps:
     commands:
       - "srun julia -O0 --color=yes --project -e 'using Pkg; Pkg.test()'"
     agents:
-      slurm_mem: 8G
+      slurm_mem: 50G
```

Comment:
👀

Reply:
we can probably reduce the memory usage of the tests right? I think often a bigger grid is used than needed

Reply:
Right, unit tests do not require too much memory. I have seen that 32G was not enough for the regression tests on the GPU.